benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadClient.java

```diff
@@ -403,11 +403,13 @@ public void run() {
               @Override
               public void onNext(Messages.SimpleResponse value) {
                 delay(System.nanoTime() - now);
```

Comment:
`now` is initialized to `nanoTime()`, and the first message is sent outside of the StreamObserver. So it looks like it is consistent.